modules/dataframe/src/io/csvreader.cpp

```diff
@@ -510,6 +524,24 @@ std::shared_ptr<DataFrame> CSVReader::readData(std::istream& stream) const {
         std::remove_if(rows.begin(), rows.end(), [](const auto& row) { return row.first.empty(); }),
         rows.end());
 
+    // Ignore comment lines
+    if (!rowComment_.empty()) {
+        rows.erase(std::remove_if(rows.begin(), rows.end(),
```

Comment:
It should be pointed out that the comment _must_ start in the very first column (no leading spaces).

Reply:
Also mention that comments are removed _before_ keep lines is applied.